.travis.yml

```diff
@@ -95,10 +39,8 @@ before_install:
 
 install:
   - travis_retry composer install $COMPOSER_ARGS --ignore-platform-reqs
```

Comment:
Please remove also `--ignore-platform-reqs`

Reply:
Why? Having that in place has opened us up for issues a number of times. What if a future release of PHPUnit only works on 7.2, for example?

Reply:
Oh, wait, never mind; I thought I'd removed it!